README.md

```diff
@@ -2,6 +2,22 @@
 
 ## Install
 
+### Docker
+
+If docker is available on your system, you can immediately get started using
+e.g.
+```
+docker run -v ${DATA_DIR_HOST}:/data ratschlab/metagraph build -v -k 10 /data/transcripts_1000.fa -o /data/transcripts_1000
```

Comment:
The idea is, that the official image is called `ratschlab/metagraph` and that the image built locally `metagraph` to avoid confusion (of course users can retag the image names locally as they see fit)

Reply:
What about `metagraph` and `metagraph_dev`/`metagraph_local`?

I think the majority won't have a need to build it locally (is this true?) so the shorter name would be better.
What do you think?

Reply:
BTW, the `ratschlab/metagraph` name comes from the naming convention on dockerhub.com

Reply:
Yeah, might be, I am not familiar with that. I'm just thinking about what would be the most sensible.